sources/osg/State.js

```diff
@@ -55,6 +55,9 @@ define( [
         // texture manager is referenced here because it's associated with gl object
         // of the gl context intialized with State
         this._textureManager = new TextureManager();
+
+
+        this._programCommonUniformsCache = {};
```

Comment:
retested but yeah in this context Map is a bit slower. But it's a special case where we dont have a lot of entry in the cache